app/routes/products_.$productId/route.tsx

```diff
@@ -66,9 +66,28 @@ export default function ProductDetailsPage() {
                 className={styles.left}
             />
             <div className={styles.right}>
-                <div>{product.name}</div>
+                <div>
+                    <div className={styles.productName}>{product.name}</div>
+                    {product.sku !== undefined && (
+                        <div className={styles.sku}>SKU: {product.sku}</div>
```

Comment:
should we add translation file later ?

Reply:
not sure, there were no such task, at the moment all content is hardcoded in this template